### DISCUSSION TITLE
host k8snode-01 has flags noout

### DISCUSSION THREAD

**Discussion by vertexvaar** [rook/rook#10628] [Q&A] [answered]
(Probably) beginner question:

Running `ceph health detail` i get the output:
```
[WRN] OSD_FLAGS: 1 OSDs or CRUSH {nodes, device-classes} have {NOUP,NODOWN,NOIN,NOOUT} flags set
    host k8snode-01 has flags noout
```

Q: How do i clear the host flag?

I ran `ceph osd unset noout` but as i would expect it, it only unsets the flags on OSDs, not hosts. As far as i see there is no `ceph node unset <flag>` and i can not edit the host in the host list because there is no checkbox to select and edit it.

**Comment by parth-gr** [comment, marked as answer]
Hey, so you can try `ceph osd unset-group noout prod-ceph-data1701` for the entire CRUSH bucket removal from `noout` flag,

Fore more references: https://docs.ceph.com/en/quincy/rados/troubleshooting/troubleshooting-osd/#stopping-w-out-rebalancing
